src/main/java/online/book/store/repository/BookRepository.java

```diff
@@ -2,8 +2,6 @@
 
 import online.book.store.model.Book;
 import org.springframework.data.jpa.repository.JpaRepository;
-import org.springframework.stereotype.Repository;
 
-@Repository
```

Comment:
Why did you remove this annotation?

Reply:
My bad!
I thought that I added this change in the previous PR before the merge.
Because when we use JpaRepository interface, then annotation @Repository is not necessary
Approved by mentor's comment from previous PR:
"It's not necessary for interfaces which extend Springs JPA related interfaces like JpaRepository and Spring applies it by default."